packages/expo-print/ios/ExpoPrintExceptions.swift

```diff
@@ -0,0 +1,67 @@
+import ExpoModulesCore
+
+internal class PrinterPickerException: GenericException<String?> {
+  override var reason: String {
+    "There was a problem with the printer picker: '\(param ?? "nil")'"
+  }
+}
+
+internal class ViewControllerNotFoundException: Exception {
+  override var reason: String {
+    "Could not find the current view controller."
+  }
+}
+
+internal class PickerCancelledException: Exception {
```

Comment:
```suggestion
internal class PickerCanceledException: Exception {
```
We prefere the American version of that word.